opennms-webapp-rest/src/test/resources/v1/nodes.json

```diff
@@ -79,6 +79,7 @@
             "sysObjectId": ".1.3.6.1.4.1.8072.3.2.255",
             "lastEgressFlow" : null,
             "lastIngressFlow" : null,
+            "nodeParentID" : null,
```

Comment:
Is this needed

Reply:
yes ,it is being used in the following test case .It was giving error unexpected nodeParentID ,after adding here it resolved.


    @Test
    @JUnitTemporaryDatabase
    public void testAnotherNodeJson() throws Exception {